### ISSUE TITLE
Add more user customization options, like controlling the speed of the speed

### ISSUE THREAD

**Issue by IamMU** [mrpandya/whatsapp-bomber#18]
## Features to Add
  - Speed Control (Almost all)
  - Speed Recommendation ( Use BeautifulSoup and Requests[ or any other library, that doesn't require opening the browser ] and use a network tester to get a speed, and recommend if user should choose fast, balanced or safe speed modes but still allow users to change it!)
  - Improve color theme
  - Add a main menu
      - Start Bot
      - Customization Options
      - Credits
      - Exit
  - Use system theme to determine which color theme to use( Light/Dark )

**Comment by mrpandya** [comment]
while adding the speed control mode, you need to take care that whatsapp does not block then bot. 
you can use the existing apis to send messages via HTTP requests to make it faster, that will also make the bot termux compatible. which is mentioned in the issue #2.

**Comment by IamMU** [comment]
Yes, the user won't be allowed to go below a certain threshold. 
About converting the bot to use HTTP requests instead of selenium, I can't help that much, as I am very bad at making the right requests. 
I can add a main menu and improve UI today, and make it use system theme.

**Comment by mrpandya** [comment]
you can create a menu and keep an alternative option for apis, we dont need to replace selenium, we can add the apis along with the selenium bot. You can create a new issue regarding the menu and also add screenshots in the readme. let this issue be open for others to work on

**Comment by IamMU** [comment]
Ok
